CHANGELOG.md

```diff
@@ -333,12 +381,7 @@ A big thanks to the 5 contributors who made this release possible. Here are some
 - 🙈 Add a new option to hide the amount of descendant on the grouping cells of the Tree Data (#3368) @flaviendelangle
 
   ```tsx
-  <DataGridPro
-    treeData
-    rows={rows}
-    columns={columns}
-    groupingColDef={{ hideDescendantCount }}
-  />
+  <DataGridPro treeData rows={rows} columns={columns} groupingColDef={{ hideDescendantCount }} />
```

Comment:
I think your prettier has modified some preview changelog

Reply:
It was @m4theushw commit but yes I suppose an update on Prettier has changed some of the behaviors